prep-for-data-groups-module/arrays-practice.js

```diff
@@ -0,0 +1,20 @@
+// write a function which takes an array as a parameter and swaps the first element with the last element
+const arr = [5, 2, 3, 4, 1];
+function swapFirstAndLast(arr){
+  // set this position one variable to hold the first element
+let positionOne = arr[0];
```

Comment:
we should try to use const whenever we don't need to change a variable after declaration
```suggestion
const positionOne = arr[0];
```

Reply:
I will definitely put it in practice. Thanks for the feedback.